### PR TITLE
FP SK1236 - Fix nova

### PR DESCRIPTION
**Pull request by fpillotovh**

### CODE REVIEW

File: pages/public_cloud/compute/prepare_the_environment_for_using_the_openstack_api/guide.en-gb.md
Comment (yomovh) [inline pages/public_cloud/compute/prepare_the_environment_for_using_the_openstack_api/guide.en-gb.md:1]: Python >=3.8 is required (see https://pypi.org/project/python-openstackclient/)

File: pages/public_cloud/compute/loading_openstack_environment_variables/guide.en-gb.md
Comment (yomovh) [inline pages/public_cloud/compute/loading_openstack_environment_variables/guide.en-gb.md:1]: Remove the /train that points to an old release

**Review by yomovh — CHANGES_REQUESTED** [review]
See comments : 2 things to update (python version on Windows + links to https://docs.openstack.org/

additionally, please add something like : 

if you need to configure the Public Cloud Load Balancer you will to run : 

pip3 install python-octaviaclient

Thanks

**Review by yomovh — APPROVED** [review]
OK for me

**Review by bs-ovh — APPROVED** [review]
rdy

File: pages/public_cloud/compute/prepare_the_environment_for_using_the_openstack_api/guide.fr-fr.md [hunk 1]
```diff
@@ -18,6 +23,12 @@ Grâce à l'API OpenStack, vous pouvez automatiser cette gestion en générant d
 
 ## En pratique
 
+To configure the Public Cloud Load Balancer you will need to install the Python Octavia client:
```
Comment (BrenKt) [inline pages/public_cloud/compute/prepare_the_environment_for_using_the_openstack_api/guide.fr-fr.md]:
```suggestion
Pour configurer le Load Balancer Public Cloud , vous devez installer le client Python Octavia :
```

**Review by BrenKt — COMMENTED** [review]
ok

**Review by BrenKt — APPROVED** [review]

**Comment by tcpdumpfbacke** [comment]
Thank you @fpillotovh !

**Pull request merged**